[
    {
        "message": "Module error while trying to sync data with repo: git command failed: Command '['git', 'pull', 'origin', 'main']' returned non-zero exit status 1. (CalledProcessError).",
        "severity": 2,
        "origin": "data_uploader.py:9 --> project.py/sync_dir_with_repo:257",
        "origin_timestamp": "2024-09-23-12-00-04",
        "registration_timestamp": "2024-09-23-12-00-04",
        "reported": true,
        "reported_timestamp": "2024-09-23-16-00-02",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to update local copy of weekly cycle for Oktopusz.: couldn't export 2D array to CSV at config/local_scheduling_files/weekly_cycle_Oktopusz.csv: 'charmap' codec can't encode character '\\u0151' in position 5: character maps to <undefined> (UnicodeEncodeError).",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/update_local_scheduling_files:53 --> project.py/export_2D_array_to_csv:1106",
        "origin_timestamp": "2024-10-05-14-00-44",
        "registration_timestamp": "2024-10-05-14-00-44",
        "reported": true,
        "reported_timestamp": "2024-10-05-19-10-19",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to update room overrides.: couldn't export 2D array to CSV at config/local_scheduling_files/override_rooms.csv: 'charmap' codec can't encode character '\\u0151' in position 53: character maps to <undefined> (UnicodeEncodeError).",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/update_local_scheduling_files:70 --> project.py/export_2D_array_to_csv:1106",
        "origin_timestamp": "2024-10-05-14-01-05",
        "registration_timestamp": "2024-10-05-14-01-05",
        "reported": true,
        "reported_timestamp": "2024-10-05-19-10-21",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to update cycle overrides.: couldn't export 2D array to CSV at config/local_scheduling_files/override_cycles.csv: 'charmap' codec can't encode character '\\u0151' in position 60: character maps to <undefined> (UnicodeEncodeError).",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/update_local_scheduling_files:87 --> project.py/export_2D_array_to_csv:1106",
        "origin_timestamp": "2024-10-05-14-01-06",
        "registration_timestamp": "2024-10-05-14-01-06",
        "reported": true,
        "reported_timestamp": "2024-10-05-19-10-24",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to update local copy of weekly cycle for vend\u00e9gt\u00e9r.: couldn't download csv from https://docs.google.com/spreadsheets/d/1OmcLOB4OvFqmVw6Jq1lxu5fo7FchPBNMM6MFoNgM7r4/pub?output=csv&2020257900:  (KeyboardInterrupt).",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/update_local_scheduling_files:53 --> project.py/download_csv_to_2D_array:1014",
        "origin_timestamp": "2024-10-05-14-01-42",
        "registration_timestamp": "2024-10-05-14-01-42",
        "reported": true,
        "reported_timestamp": "2024-10-05-19-10-27",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to update room overrides.: couldn't download csv from https://docs.google.com/spreadsheets/d/1Cir4TYYhIx14Qja9c4nN9m2GTeZ0FHrJRt03oXs_fgI/pub?output=csv:  (KeyboardInterrupt).",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/update_local_scheduling_files:70 --> project.py/download_csv_to_2D_array:1014",
        "origin_timestamp": "2024-10-05-14-01-44",
        "registration_timestamp": "2024-10-05-14-01-44",
        "reported": true,
        "reported_timestamp": "2024-10-05-19-10-28",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to update cycle overrides.: couldn't download csv from https://docs.google.com/spreadsheets/d/1U0QnnC-E0hrbp0pf5foDVTwHQTmSteIG8vXHI6MXkh8/pub?output=csv:  (KeyboardInterrupt).",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/update_local_scheduling_files:87 --> project.py/download_csv_to_2D_array:1014",
        "origin_timestamp": "2024-10-05-14-01-44",
        "registration_timestamp": "2024-10-05-14-01-44",
        "reported": true,
        "reported_timestamp": "2024-10-05-19-10-30",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Unexpected error while trying to update room overrides.: cannot access local variable 'response' where it is not associated with a value.",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/update_local_scheduling_files:78 --> project.py/write:253 --> project.py/interact:232",
        "origin_timestamp": "2024-10-05-14-30-28",
        "registration_timestamp": "2024-10-05-14-30-28",
        "reported": true,
        "reported_timestamp": "2024-10-05-19-10-32",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to update local copy of weekly_cycle/1.: couldn't export 2D array to CSV at config/local_scheduling_files/weekly_cycle_Oktopusz.csv: [Errno 2] No such file or directory: 'C:\\\\Users\\\\Beno\\\\Documents\\\\SZAKI\\\\dev\\\\kazanfutes\\\\config/local_scheduling_files/weekly_cycle_Oktopusz.csv' (FileNotFoundError).",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/update_local_scheduling_files:77 --> project.py/export_2D_array_to_csv:1138",
        "origin_timestamp": "2024-10-05-15-48-57",
        "registration_timestamp": "2024-10-05-15-48-57",
        "reported": true,
        "reported_timestamp": "2024-10-05-19-10-35",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to update local copy of weekly_cycle/4.: couldn't download csv from https://docs.google.com/spreadsheets/d/1OmcLOB4OvFqmVw6Jq1lxu5fo7FchPBNMM6MFoNgM7r4/pub?output=csv&gid=228406814:  (KeyboardInterrupt).",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/update_local_scheduling_files:77 --> project.py/download_csv_to_2D_array:1046",
        "origin_timestamp": "2024-10-05-15-49-01",
        "registration_timestamp": "2024-10-05-15-49-01",
        "reported": true,
        "reported_timestamp": "2024-10-05-19-10-38",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to update local copy of weekly_cycle/G\u00f3lyairoda.: couldn't download csv from None: Invalid URL 'None': No scheme supplied. Perhaps you meant https://None? (MissingSchema).",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/update_local_scheduling_files:90 --> project.py/download_csv_to_2D_array:1055",
        "origin_timestamp": "2024-10-05-18-21-25",
        "registration_timestamp": "2024-10-05-18-21-25",
        "reported": true,
        "reported_timestamp": "2024-10-05-19-10-41",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Unexpected error while trying to generate condensed schedule.: 'list' object has no attribute 'join'.",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/generate_and_export_condensed_schedule:169",
        "origin_timestamp": "2024-10-05-18-50-16",
        "registration_timestamp": "2024-10-05-18-50-16",
        "reported": true,
        "reported_timestamp": "2024-10-05-19-10-45",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Unexpected error while trying to generate condensed schedule.: 'list' object has no attribute 'join'.",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/generate_and_export_condensed_schedule:170",
        "origin_timestamp": "2024-10-05-18-52-38",
        "registration_timestamp": "2024-10-05-18-52-38",
        "reported": true,
        "reported_timestamp": "2024-10-05-19-10-46",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Unexpected error while trying to generate condensed schedule.: 'list' object has no attribute 'join'.",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/generate_and_export_condensed_schedule:171",
        "origin_timestamp": "2024-10-05-18-56-17",
        "registration_timestamp": "2024-10-05-18-56-17",
        "reported": true,
        "reported_timestamp": "2024-10-05-19-10-47",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to generate condensed schedule.: couldn't load csv from config/local_scheduling_files/weekly_cycle_Lahmacun.csv:  (KeyboardInterrupt).",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/generate_and_export_condensed_schedule:149 --> project.py/load_csv_to_2D_array:1068",
        "origin_timestamp": "2024-10-05-18-56-30",
        "registration_timestamp": "2024-10-05-18-56-30",
        "reported": true,
        "reported_timestamp": "2024-10-05-19-10-50",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Unexpected error while trying to update local heating control config: 'heating_control_config_url'.",
        "severity": 2,
        "origin": "schedule_and_ config_updater.py/update_local_heating_control_config:42",
        "origin_timestamp": "2024-10-05-19-34-14",
        "registration_timestamp": "2024-10-05-19-34-14",
        "reported": true,
        "reported_timestamp": "2024-10-06-01-08-41",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Unexpected error while trying to update local heating control config: 'heating_control_config_url'.",
        "severity": 2,
        "origin": "schedule_and_ config_updater.py/update_local_heating_control_config:46",
        "origin_timestamp": "2024-10-05-19-52-47",
        "registration_timestamp": "2024-10-05-19-52-47",
        "reported": true,
        "reported_timestamp": "2024-10-06-01-08-41",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Unexpected error while trying to initialize heating control: strptime() takes exactly 2 arguments (1 given).",
        "severity": 3,
        "origin": "heating_control.py/compare_and_command:56 --> project.py/get_room_temps_and_humidity_dev:703",
        "origin_timestamp": "2024-10-05-20-37-04",
        "registration_timestamp": "2024-10-05-20-37-04",
        "reported": true,
        "reported_timestamp": "2024-10-06-01-08-41",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Unexpected error while trying to initialize heating control: strptime() takes exactly 2 arguments (1 given).",
        "severity": 3,
        "origin": "heating_control.py/compare_and_command:57 --> project.py/get_room_temps_and_humidity_dev:703",
        "origin_timestamp": "2024-10-05-20-37-18",
        "registration_timestamp": "2024-10-05-20-37-18",
        "reported": true,
        "reported_timestamp": "2024-10-06-01-08-41",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to update local heating control config: couldn't download csv from https://docs.google.com/spreadsheets/d/e/2PACX-1vQesA7RtQho9HwcVSOitNyLRDn6oODG5K7-s6C85kWOLwcz7yiGg2d_SuBYhoCkDAVUQ0vXGzaa12dn/pub?output=csv:  (KeyboardInterrupt).",
        "severity": 2,
        "origin": "schedule_and_ config_updater.py/update_local_heating_control_config:48 --> project.py/download_csv_to_2D_array:1081",
        "origin_timestamp": "2024-10-05-22-39-29",
        "registration_timestamp": "2024-10-05-22-39-29",
        "reported": true,
        "reported_timestamp": "2024-10-06-01-08-41",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to update local copy of weekly_cycle/1: couldn't download csv from https://docs.google.com/spreadsheets/d/1OmcLOB4OvFqmVw6Jq1lxu5fo7FchPBNMM6MFoNgM7r4/pub?output=csv&gid=1887663707:  (KeyboardInterrupt).",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/update_local_scheduling_files:129 --> project.py/download_csv_to_2D_array:1081",
        "origin_timestamp": "2024-10-05-22-39-32",
        "registration_timestamp": "2024-10-05-22-39-32",
        "reported": true,
        "reported_timestamp": "2024-10-06-01-08-41",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to update local copy of weekly_cycle/1: couldn't export 2D array to CSV at config/local_scheduling_files/weekly_cycle_room_{'name': 'Oktopusz', 'sensor': None, 'cycle': '1', 'schedule_gid': '1887663707'}.csv: [Errno 22] Invalid argument: \"C:\\\\Users\\\\Beno\\\\Documents\\\\SZAKI\\\\dev\\\\kazanfutes\\\\config/local_scheduling_files/weekly_cycle_room_{'name': 'Oktopusz', 'sensor': None, 'cycle': '1', 'schedule_gid': '1887663707'}.csv\" (OSError).",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/update_local_scheduling_files:129 --> project.py/export_2D_array_to_csv:1297",
        "origin_timestamp": "2024-10-06-19-22-33",
        "registration_timestamp": "2024-10-06-19-22-33",
        "reported": true,
        "reported_timestamp": "2024-10-07-00-00-02",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to generate condensed schedule: couldn't load csv from config/local_scheduling_files/weekly_cycle_room_{'name': 'Oktopusz', 'sensor': None, 'cycle': '1', 'schedule_gid': '1887663707'}.csv: [Errno 22] Invalid argument: \"C:\\\\Users\\\\Beno\\\\Documents\\\\SZAKI\\\\dev\\\\kazanfutes\\\\config/local_scheduling_files/weekly_cycle_room_{'name': 'Oktopusz', 'sensor': None, 'cycle': '1', 'schedule_gid': '1887663707'}.csv\" (OSError).",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/generate_condensed_schedule:183 --> project.py/load_csv_to_2D_array:1210",
        "origin_timestamp": "2024-10-06-19-22-33",
        "registration_timestamp": "2024-10-06-19-22-33",
        "reported": true,
        "reported_timestamp": "2024-10-07-00-00-02",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to update local copy of weekly_cycle/8: couldn't download csv from https://docs.google.com/spreadsheets/d/1OmcLOB4OvFqmVw6Jq1lxu5fo7FchPBNMM6MFoNgM7r4/pub?output=csv&gid=60517023:  (KeyboardInterrupt).",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/update_local_scheduling_files:129 --> project.py/download_csv_to_2D_array:1197",
        "origin_timestamp": "2024-10-06-19-22-33",
        "registration_timestamp": "2024-10-06-19-22-33",
        "reported": true,
        "reported_timestamp": "2024-10-07-00-00-02",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to acquire temperature and humidity data: unexpected error while reading sensor state: time data 'none' does not match format '%Y-%m-%dT%H:%M:%S.%f' (ValueError).",
        "severity": 3,
        "origin": "temperature_and_humidity_logger.py:10 --> project.py/get_room_temps_and_humidity:734",
        "origin_timestamp": "2024-10-10-15-25-01",
        "registration_timestamp": "2024-10-10-15-25-01",
        "reported": true,
        "reported_timestamp": "2024-10-10-16-00-02",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to acquire temperature and humidity data: unexpected error while reading sensor state: 'Multiple conflicting time zone configurations found:\\n/etc/timezone: Europe/London\\n/etc/localtime is a symlink to: Europe/Budapest\\nFix the configuration, or set the time zone in a TZ environment variable.\\n' (ZoneInfoNotFoundError).",
        "severity": 3,
        "origin": "temperature_and_humidity_logger.py:10 --> project.py/get_room_temps_and_humidity:741",
        "origin_timestamp": "2024-10-11-14-05-02",
        "registration_timestamp": "2024-10-11-14-05-02",
        "reported": true,
        "reported_timestamp": "2024-10-11-16-00-03",
        "checked": false,
        "checked_timestamp": null
    },
    {
        "message": "Module error while trying to update local copy of weekly_cycle/4: couldn't download csv from https://docs.google.com/spreadsheets/d/1OmcLOB4OvFqmVw6Jq1lxu5fo7FchPBNMM6MFoNgM7r4/pub?output=csv&gid=228406814: ('Connection aborted.', ConnectionAbortedError(10053, 'An established connection was aborted by the software in your host machine', None, 10053, None)) (ConnectionError).",
        "severity": 3,
        "origin": "schedule_and_ config_updater.py/update_local_scheduling_files:131 --> project.py/download_csv_to_2D_array:1497",
        "origin_timestamp": "2024-10-14-17-08-39",
        "registration_timestamp": "2024-10-14-17-08-39",
        "reported": false,
        "reported_timestamp": null,
        "checked": false,
        "checked_timestamp": null
    }
]